lua/entities/gmod_wire_expression2/core/player.lua

```diff
@@ -332,6 +332,31 @@ e2function string keyClkPressed()
 end
 
 
+-- Use Support --
+
+local InUseBy = nil
+__e2setcost(50)
+--- Makes the chip "Use"able
+e2function void runOnUse(value)
+	if value != 0 then
+		self.entity:SetUseType( SIMPLE_USE )
+		self.entity.Use = function(self,activator)
+			InUseBy = activator
+			self:Execute()
+			InUseBy = nil
```

Comment:
Shouldn't this return a NULL entity instead of nil for better compatibility?

or on Line 356 add in a `InUseBy or NULL`?

Reply:
Yes. Use `NULL` instead of `nil`.

Reply:
I think there were some changes to how Execute is done or something recently too? Something related to stack or whatever